composer.json

```diff
@@ -23,33 +23,35 @@
         }
     ],
     "require": {
-        "php": "^5.3.3|^7.0",
-        "doctrine/common": "^2.3|^3.0",
-        "symfony/config": "^2.3|^3.0",
-        "symfony/http-kernel": "^2.3|^3.0",
-        "symfony/http-foundation": "^2.4|^3.0",
-        "symfony/dependency-injection": "^2.3|^3.0"
+        "php": "^5.4|^7.0",
+        "doctrine/common": "^2.5",
+        "symfony/config": "^2.8|^3.0",
```

Comment:
Well, I'm always a fan of using the most recent version, but do you really want to kick out those guys, who still uses 2.3/2.5/2.7? 
`2.3` will be supported until 2017 and `2.7` until 2019. 
Maybe it would be a good idea to maintain a bug-fix-only-version for 1.x which supports symfony < 2.8?